source/installation/installation.rst

```diff
@@ -220,13 +224,13 @@ Populate your database with basic information
 
 Once your database schema is ready, you should populate your database with some basic information. Those are provided through scripts and might depends from the bundle you choose to install (see :ref:`install-additional-bundles`)
 
-The main bundle require two scripts to be executed : 
+The main bundle require one script to be executed : 
```

Comment:
This command is valid for all bundles. I suggest to write "To install those fixtures (dummy data), you should execute : "